### ISSUE TITLE
Orders still reading unpaid even after payment 

### ISSUE THREAD

**Issue by gathia** [Collinslenjo/LipaNaMpesa#1]
I uploaded it on my site and did the settings for the API, but if i make a pament the order status still read not paid, even for virtual products on woocommerce. 
Whats the issue?

**Comment by Collinslenjo** [comment]
if you followed the installation process correctly and enabled it in Woocommerce checkout then you have to confirm  the phone Number and the code you get from Mpesa on the form that appears on the front-end checkout form.

**Comment by gathia** [comment]
Yes, i enabled it on woocommerce, and set the API but. But still not working

**Comment by Collinslenjo** [comment]
Are you confirming on the frontend. There is a form that takes the params [phonenumber and mpesa receipt]

**Comment by gathia** [comment]
on the woocomerce back end settings there are the fields for 
"Confirmation Code Field Title"
"Phone Number Field Title"

Should i leave them blank ?

On the front end the form to fill Phone number and Mpesa receipt displays but when i key in the phn no. and the transaction code the invoice does not tick as paid

Do you have a video guide?

**Comment by Collinslenjo** [comment]
There is no video guide at the moment but if you have paid it should change the status of the product to processing if not downloadable.
